uiowa/uiowa
# Fix MAUI block AJAX bug with multiple forms.

Resolves #2889 

### How to test
Reproduce the bug on main according to #2889 using grad.uiowa.edu locally. Check out this branch, maybe rebuild the cache and check if the behavior is as expected.
